src/kibana/components/config/defaults.js

```diff
@@ -11,6 +11,10 @@ define(function (require) {
       value: 'MMMM Do YYYY, HH:mm:ss.SSS',
       description: 'When displaying a pretty formatted date, use this format',
     },
+    'dateFormat:tz': {
+      value: 'browser',
+      description: 'Which timezone should be used.  (e.g. "browser", "utc", or "America/Chicago")',
+    },
```

Comment:
If you use https://github.com/elastic/kibana/pull/3638 you can replace this with:

```
    'dateFormat:tz': {
      value: 'Default',
      description: 'Which timezone should be used',
      type: 'select',
      options: _.union(['Default'], moment.tz.names())
    },
```